### ISSUE TITLE
Remove usage of legacy ELEMENT pointer origin for actions

### ISSUE THREAD

**Issue by whimboo** [SeleniumHQ/selenium#6393]
Geckodriver currently [needs a fallback](https://bugzilla.mozilla.org/show_bug.cgi?id=1489141) for deserializing the pointer origin because at least the Java implementation is still using the legacy `ELEMENT` identifier for pointer move primitives. Here an excerpt from the trace log:

```
        {
          "duration": 100,
          "x": 0,
          "y": 0,
          "type": "pointerMove",
          "origin": {
            "ELEMENT": "cbd0b1e7-a456-4730-bf2b-b8ba7ec6aba1",
            "element-6066-11e4-a52e-4f735466cecf": "cbd0b1e7-a456-4730-bf2b-b8ba7ec6aba1"
          }
        },
```

This is not used by other bindings like Python anymore, and Java should also drop that at some point. @barancev and @shs96c, not sure when you actually want to do that?

As long as this hasn't been done we will keep the fallback in geckodriver.

**Comment by lmtierney** [comment]
As discussed in IRC, ETA of Selenium 4

**Comment by titusfortner** [comment]
Looking at how we're doing this, I'm not sure we can remove this before removing the entire handshake code. I'm running the tests so we can see - https://github.com/SeleniumHQ/selenium/actions/runs/2799798464

**Comment by titusfortner** [comment]
@diemol when I'm suggesting we don't have good OSS tests to ensure handshake is working... I'm pretty sure this branch should have a bunch of failures, but everything is passing.

**Comment by diemol** [comment]
This was implemented by #11823
